win32/Lib/winnt.py

```diff
@@ -161,34 +161,22 @@
 
 
 def PRIMARYLANGID(lgid):
-    return (WORD)(lgid) & 1023
+    return (lgid) & 1023
 
 
 def SUBLANGID(lgid):
-    return (WORD)(lgid) >> 10
```

Comment:
`(WORD)` and `(DWORD)` seem like invalid leftovers from C casting transpilation.

Same as `win32con.py`